meson.build

```diff
@@ -16,6 +16,8 @@ graphene_micro_version = version_array[2].to_int()
 
 graphene_api_version = '@0@.0'.format(graphene_major_version)
 
+graphene_string = '@0@-@1@'.format(meson.project_name(), graphene_api_version)
```

Comment:
Maybe this could have a better name; `graphene_string` is a bit opaque.

What about `graphene_api_string`, instead?